### ISSUE TITLE
read.js empty when npm install

### ISSUE THREAD

**Issue by AurelienMendes** [transcend-io/conflux#39]
When I follow your indications (npm install --save @transcend-io/conflux) I get an empty read.js file (actually there is only the line // TODO: later inside)

Do you understand why ?

**Comment by skyfore** [comment]
in "version": "0.0.1",  `write.js` code can't be used

```js
class Writer extends TransformStream {
  #files = Object.create(null)
  #filenames = []
  #offset = 0

  constructor () {
    super({
```

**Comment by AurelienMendes** [comment]
Thanks for your reply @skyfore ... but I was talking about the _read.js_ since I'm just interested in reading huge Zip files ...

**Comment by AurelienMendes** [comment]
Hey ... what I've done is to download the master branch (with the good read.js file) and use it for tests in my project.
I clone the _example/reading.html_ (same code, same cdn files, ...) but got errors: 
![image](https://user-images.githubusercontent.com/44461190/74519923-ed758c00-4f16-11ea-87e5-076d45e307a7.png)

Can someone confirm me that the library works well ? And if yes, can someone explain me how to use it to just read a zip file ? Does it possible with version _0.0.1_ or do I use the _master_ ?

Thanks

**Comment by bencmbrook** [comment]
Hey @AurelienMendes and @skyfore! `read.js` has been fixed up and published on npm under version `1.1.1`. The library works well, but we're keeping it flagged as experimental for the next few months.

"Cannot read property '28789' .." was a bug fixed in #45. Thanks for reporting it.

### Using `read.js`:
- Docs https://github.com/transcend-io/conflux#reading-zip-files
- Check out https://conflux.netlify.com/example/reading for an updated example (note that `read.js` doesn't yet have support for zip64 files.)

**Comment by bencmbrook** [comment]
@AurelienMendes and @skyfore I'm curious if you got up and running - any feedback would be appreciated :)
